scripts/resource/class_generator.py

```diff
@@ -392,136 +392,84 @@ def parse_explain(
     debug_content: Optional[Dict[str, str]] = None,
     add_tests: bool = False,
 ) -> Dict[str, Any]:
-    section_data: str = ""
     sections: List[str] = []
     resource_dict: Dict[str, Any] = {
         "BASE_CLASS": "NamespacedResource" if namespaced else "Resource",
     }
-    new_sections_words: Tuple[str, str, str] = ("KIND:", "VERSION:", "GROUP:")
 
-    for line in output.splitlines():
-        # If line is empty section is done
-        if not line.strip():
-            if section_data:
-                sections.append(section_data)
-                section_data = ""
+    raw_resource_dict: Dict[str, str] = {}
 
-            continue
+    # Get all sections from output, section is [A-Z]: for example `KIND:`
+    sections = re.findall(r"([A-Z]+:).*", output)
```

Comment:
will moving ":" outside the group remove the need to strip it?
`re.findall(r"([A-Z]+):.*", output)`